scripts/local.js

```diff
@@ -0,0 +1,60 @@
+const webdriver = require('selenium-webdriver');
+const { By } = require('selenium-webdriver');
+const assert = require('assert');
+var browserstack = require('browserstack-local');
+
+//creates an instance of Local
+var bs_local = new browserstack.Local();
+
+// replace <browserstack-accesskey> with your key. You can also set an environment variable - "BROWSERSTACK_ACCESS_KEY".
+var bs_local_args = { 'key': ACCESS_KEY };
+
+// starts the Local instance with the required arguments
+bs_local.start(bs_local_args, function() {
+  console.log("Started BrowserStackLocal");
+  // check if BrowserStack local instance is running
+  console.log(bs_local.isRunning());
+  runTestWithCaps(); 
+});
+// Input capabilities
+var capabilities = {
```

Comment:
Let's move all configs to config files rather than have users make changes in multiple files, so we won't have caps in each file for local, single and parallel.